src/Uno.UI.Runtime.Skia.Wpf/UI/Controls/UnoWpfWindow.cs

```diff
@@ -114,4 +121,27 @@ internal void UpdateWindowPropertiesFromPackage()
 			Title = Windows.ApplicationModel.Package.Current.DisplayName;
 		}
 	}
+
+	internal void UpdateWindowPropertiesFromCoreApplication()
+	{
+		var coreApplicationView = CoreApplication.GetCurrentView();
+		if (coreApplicationView.TitleBar.ExtendViewIntoTitleBar)
+		{
+			WindowStyle = System.Windows.WindowStyle.None;
+			WindowChrome.SetWindowChrome(this, new WindowChrome
+			{
+				// this removes the thin white bar at the top, but this causes the window to grow a little.
+				// No work around was found for this yet.
+				CaptionHeight = 0
```

Comment:
Will this mean the window can not be dragged by default?

Reply:
Yes, there is nothing to hold on to, so dragging with a mouse won't be available. This is similar to what GTK does when there are no decorations.